content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst

```diff
@@ -2,99 +2,135 @@
+
+To create a new warehouse, navigate to :menuselection:`Inventory app --> Configuration -->
+Warehouses`, and click :guilabel:`New`. Doing so opens a blank warehouse creation form.
+
+Fill out the following fields as necessary:
+
+- :guilabel:`Warehouse`: the full name of the warehouse.
+- :guilabel:`Short Name`: the abbreviated code used to identify the warehouse, i.e. `WH2` (the short
+  name for the default warehouse in Odoo is `WH`).
+- :guilabel:`Company`: the company that owns the warehouse. This can be set as the company that owns
+  the Odoo database or the company of a customer or vendor.
+- :guilabel:`Address`: the address where the warehouse is located. By default, this is set to the
+  company's address in the database.
+
+.. image:: create_a_second_warehouse/create-a-second-warehouse-new-warehouse-settings.png
    :align: center
-   :alt: A filled out form for creating a new warehouse.
+   :alt: Filled out warehouse name and address settings on warehouse creation form.
+
+Under the :guilabel:`Warehouse Configuration` tab, edit the :guilabel:`Shipments` and
+:guilabel:`Resupply` settings as necessary.
 
 .. important::
-   Creating a second warehouse will automatically enable the *Storage Locations* setting, which
-   allows location tracking of products within a warehouse. To toggle this setting, navigate to
-   :menuselection:`Configuration --> Settings` and click the checkbox under the
-   :guilabel:`Warehouse` heading.
+   The :guilabel:`Shipments` and :guilabel:`Resupply` sections only appear if the
+   :guilabel:`Multi-Step Routes` checkbox is enabled in :menuselection:`Inventory app -->
+   Configuration --> Settings`, under the :guilabel:`Warehouse` heading. For more information about
+   routes and how they work in Odoo, see :ref:`Using Routes and Pull/Push Rules <use-routes>`.
+
+In the :guilabel:`Shipments` section, edit the following settings:
+
+- :guilabel:`Incoming Shipments`: default incoming route to follow. Choose from one, two, and
+  three-step receipts.
+- :guilabel:`Outgoing Shipments`: default outgoing route to follow. Choose from one, two, and
+  three-step deliveries.
+
+In the :guilabel:`Resupply` section, edit the following settings:
+
+- :guilabel:`Dropship Subcontractors`: dropship subcontractors with components.
+- :guilabel:`Resupply Subcontractors`: allow subcontractors to be resupplied with goods from this
+  warehouse.
+- :guilabel:`Manufacture to Resupply`: when products are manufactured, they can be manufactured in
+  this warehouse.
+- :guilabel:`Manufacture`: the route that should be followed when manufacturing goods inside
+  the warehouse. Choose from one, two, and three-step production.
+- :guilabel:`Buy to Resupply`: when products are purchased, they can be delivered to this warehouse.
+- :guilabel:`Resupply From`: tick the checkbox next to the warehouse that this warehouse should be
+  resupplied from. Resupply routes are automatically created for the selected warehouse.
```

Comment:
I'm not familiar with the resupply from another warehouse route, so this section was confusing to me. Given that the resupply route is only available as a field when there's more than one warehouse created in a database, I think it's important to expand on this field a little more.

(For example, when I only have 1 WH, the "Resupply From" field isn't there:
![image](https://github.com/odoo/documentation/assets/23611711/23b824b5-37f4-4590-b78f-f45eb0e7d019)
 )
 
 Can we possibly:
 - elaborate a little more what a "warehouse that this warehouse should be resupplied from", means?
   - e.g. "Check the box next to the **other** warehouses from which **this** warehouse can replenish products. If the current warehouse does not have the stock, it can also pull products from these selected warehouses in the database. Automatic resupply routes will be created accordingly."
 - link to the [resupply WH doc](https://www.odoo.com/documentation/17.0/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/resupply_warehouses.html) for more info

Reply:
Suggestion implemented ✅